pages/public_cloud/containers_orchestration/managed_kubernetes/expose_your_applications_using_a_load_balancer/guide.en-gb.md

```diff
@@ -419,11 +427,104 @@ kubectl apply -f your-service-manifest.yaml
+
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  name: octavia-keepip-with-existing-ip
+  annotations:
+    loadbalancer.ovhcloud.com/class: "octavia"
+    #loadbalancer.openstack.org/keep-floatingip: "true" # Useless, since the FIP was provided, the FIP will not be managed by the MKS cluster
+spec:
+  loadBalancerIP: 1.2.3.4
+  type: LoadBalancer
+```
+
+> [!primary]
+>
+> Note: the FloatingIP must be in the same region as the MKS cluster. A FloatingIP cannot be moved to another OpenStack region.
+>
+
+### Use a fixed Virtual-IP (VIP) for the LoadBalancer in the subnet
+
+To assign a fixed VIP to the LoadBalancer in the OpenStack subnet, you have to create an OpenStack Port. e.g. with the OpenStack CLI:
+
+```shell
+openstack port create --network <network> --fixed-ip subnet=<subnet>,ip-address=10.2.3.4 my-fixed-octavia-vip
+```
+
+Then use the annotation `loadbalancer.openstack.org/port-id` with the OpenStack port's UUID:
+
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  name: octavia-with-fixed-vip
+  annotations:
+    loadbalancer.ovhcloud.com/class: "octavia"
+    loadbalancer.openstack.org/port-id: "<openstack-port-uuid>"
+spec:
+  type: LoadBalancer
+```
+
+> [!warning]
+>
+> This feature is only supported when the MKS cluster is attached to a private network. (Public MKS is not supported)
+> The port needs to belong to the same network/subnet as the MKS cluster.
+> This feature is not compatible when the MKS cluster is configured with a LoadBalancerSubnetId (except if is the same as the NodeSubnetID).
+>
+
+### Restrict access for a LoadBalancer Service
+
+To apply IP restriction to the K8S LoadBalancer Service, you can define the array `.spec.loadBalancerSourceRanges` with a list of CIDR.
+
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  name: octavia-ip-restrictions
+  annotations:
+    loadbalancer.ovhcloud.com/class: "octavia"
+spec:
+  loadBalancerSourceRanges:
+  - 1.2.3.4/32
+  - 5.6.7.0/24
+  type: LoadBalancer
+```
+
+If no value is assigned to this spec, no restriction will be applyed.
+
+### Sharing a Octavia LoadBalancer between multiple Kubernetes LoadBalancer Services
 
-#### Sharing load balancer with multiple Services
+You can share up to two Kubernetes Services on the same Octavia LoadBalancer. Those Services can be deployed on different MKS clusters (clusters must be in the same network).
+
+K8S services must expose different protocol/port (you cannot set the same protocol/port on both K8S Services). [Sharing load balancer with multiple Services (Github)](https://github.com/kubernetes/cloud-provider-openstack/blob/master/docs/openstack-cloud-controller-manager/expose-applications-using-loadbalancer-type-service.md#sharing-load-balancer-with-multiple-services)
+
+To allow another K8S LoadBalancer Service to use an existing Octavia (created via MKS or via OpenStack), use the annotation `loadbalancer.openstack.org/load-balancer-id`:
```

Comment:
Do you think we could have an additional about "Importing an existing octavia load balancer", or do we consider it's already covered by the CCM docs?

Reply:
Wait I missed line 320, sorry